### PR TITLE
[NETBEANS-1342] Hide the panel for iOS in the Mobile Platforms Options

### PR DESCRIPTION
**Pull request by junichi11**
https://issues.apache.org/jira/browse/NETBEANS-1342

Just hide the panel because the cordova.platforms.ios module is not available (see #856)

netbeans-1342-options-panel-after

### CODE REVIEW

**Comment by Chris2011** [comment]
But when we hide it, we need to hide the cordova option inside of the HTML5/JS project too and to hide the cordova project template too.

**Comment by junichi11** [comment]
I'm not sure about cordova but I think we should remove cordova modules from the cluster if we will not support cordova yet.

File: webcommon/cordova/src/org/netbeans/modules/cordova/CordovaPerformer.java [hunk 1]
```diff
@@ -324,15 +325,16 @@ private Properties properties(Project p) {
...
-        final String codeSignIdentity = iosPlatform.getCodeSignIdentity();
-
-        if (codeSignIdentity != null) {
-            props.put(PROP_CERTIFICATE_NAME, codeSignIdentity);
+            if (codeSignIdentity != null) {
+                props.put(PROP_CERTIFICATE_NAME, codeSignIdentity);
+            }
         }
 
         //workaround for some strange behavior of ant execution in netbeans
```
Comment (sdedic) [inline webcommon/cordova/src/org/netbeans/modules/cordova/CordovaPerformer.java]: there's also a reference to `iosPlatform` at line 357; is it OK to leave unchecked ?

**Comment by junichi11** [comment]
@sdedic 
> There are also some IOS-related factories in CordovaBrowserFactory shouldn't they move to ios support module ?

Maybe, we should do so. I'm not an expert this area. So if you can do that, I'll leave it to you as another problem :) I just wanted to fix NETBEANS-1342 (NPE).

> Re. remove: I've removed the ios module from the cluster because of license issues, but there's also android support, which seemed to be license-OK.

Yes, I know it. I thought so. But I thought need not to add the cordova modules if we need to hide cordova options and templates (Chris wrote above).

**Comment by junichi11** [comment]
> CordovaPanel source defines controls related to IOS, but I can't find a place where the panel is added to a visible parent - is it a dead code ?

I'm not sure what this means, sorry... CordovaPanel is MobilePlatformsPanel?
Here?
https://github.com/apache/incubator-netbeans/blob/d428744a920fbea150be723a0c2774e384d9c334/webcommon/cordova/src/org/netbeans/modules/cordova/options/MobilePlatformsPanel.java#L81

**Comment by sdedic** [comment]
@junichi11 in project UI, here: https://github.com/apache/incubator-netbeans/blob/d428744a920fbea150be723a0c2774e384d9c334/webcommon/cordova/src/org/netbeans/modules/cordova/project/CordovaPanel.java#L79

**Comment by Chris2011** [comment]
Excluding is, in my opinion not an option as long as we don't have a good working NBAndroid Plugin, where it can live there. Hiding for the first thing is good, the rest shouldn't that hard to figure it out. I would prefer adding more feature instead of removing, otherwise, the users will find/use another IDE. IMHO. But of course this should be discussed at the mailing list.

**Comment by junichi11** [comment]
@sdedic I'm not sure about that because I'm not the author, but I could also not find it... 
It seems that just setVisible() method is called.
https://github.com/apache/incubator-netbeans/blob/4daaf04bee3a602a6a93a35422190e4ceec0bc0e/webcommon/cordova/src/org/netbeans/modules/cordova/project/CordovaPanel.java#L71-L74

Just in case, I'll hide the iOSPanel.

**Comment by junichi11** [comment]
If there is no problem, let's squash and merge. At least, this change prevents NPE, I think.

**Comment by geertjanw** [comment]
Cool, go ahead.

**Pull request merged**